core/trino-main/src/main/java/io/trino/testing/LocalQueryRunner.java

```diff
@@ -691,7 +694,10 @@ public ScheduledExecutorService getScheduler()
     @Override
     public Session getDefaultSession()
     {
-        return defaultSession;
+        return Session
```

Comment:
I don't think you should modify default session in `getDefaultSession` getter (it stops being a getter then). You should rather alter session on `io.trino.testing.LocalQueryRunner#execute(java.lang.String)`

Reply:
Moved to AbstractTestQueryFramework